### PR TITLE
Add ability to generate document history 

### PR DESCRIPTION
**Pull request by Zacqary**
Using a new `history` config option, you can tell Slingshot to generate a long history of documents without waiting for them to generate in real time. It can do this in parallel to the continuous `cycles` script.

### CODE REVIEW

File: src/lib/load.js [hunk 1]
```diff
@@ -79,7 +90,7 @@ module.exports = async function load(initialize, options) {
     } catch (err) {
       logger.error(
         `${CYCLE_NAME} Error(s) while loading documents (turn on verbose logging to see full error result): ${err.message}`
-      );
+      );``
```
Comment (jasonrhodes) [inline src/lib/load.js]: Is this a typo?

File: src/lib/load.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ const { Client } = require("./es");
 const get_logger = require("./logger");
 const dot = require("dot-object");
 const Mustache = require("mustache");
+const { option, options } = require("yargs");
```
Comment (jasonrhodes) [inline src/lib/load.js]: Are these being used? I just noticed them here but we also have "options" as the 2nd passed arg into "load' below so I don't think `yargs.options` is reachable, at least?

**Review by jasonrhodes — APPROVED** [review]
History stuff looks great! Thank you for implementing that!! 

Just had a couple quick questions, otherwise LGTM

File: src/lib/load.js (hunk 1 above)
Comment (Zacqary) [reply src/lib/load.js]: Looks like it, yeah

File: src/lib/load.js (hunk 2 above)
Comment (Zacqary) [reply src/lib/load.js]: Not sure how this got added

**Review by jasonrhodes — APPROVED** [review]
Thanks!

**Pull request merged**